### PR TITLE
Update web-based-device-enrollment-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to step8 in step 2

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4412

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/web-based-device-enrollment-ios.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Create an enrollment profile for devices enrolling via web-based device enrollme
 5. Select **Next**.  
 6. On the **Settings** page, for **Enrollment type**, select **Web based device enrollment**.  
 7. Select **Next**.  
-8. On the **Assignments** page, assign the profile to all devices, or select specific groups.
+8. On the **Assignments** page, assign the profile to all Users or a group of users.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/web-based-device-enrollment-ios.md:73]:
```suggestion
8. On the **Assignments** page, assign the profile to all users or a group of users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@lenewsad Please review and signoff this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by lenewsad** [comment]
Hi @msbemba @yogkumgit, in review with the product team.

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**